internal/server/data/schema.sql

```diff
@@ -292,6 +311,8 @@ CREATE UNIQUE INDEX idx_identities_name ON identities USING btree (organization_
 
 CREATE UNIQUE INDEX idx_identities_verified ON identities USING btree (organization_id, verification_token) WHERE (deleted_at IS NULL);
 
+CREATE UNIQUE INDEX idx_org_provider_group_names ON provider_groups USING btree (organization_id, provider_id, name);
```

Comment:
Any other index suggestions?

Reply:
every field you use in a where clause needs an index or to be part of a composite index